### PR TITLE
Support http backend override in http backend rule. 

### PR DESCRIPTION
**Pull request by shuoyang2016**
This is noop for espv2.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (TAOXUY) [inline package.json:1]: Is this file needed?

**Review by TAOXUY — COMMENTED** [review]
This PR seems a combination of sync for g3 changes like cl/517310974. Could you please put all these changes in the description for context?

**Review by nareddyt — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (shuoyang2016) [reply package.json:1]: It's auto generated when install dependencies.

**Comment by shuoyang2016** [comment]
> This PR seems a combination of sync for g3 changes like cl/517310974. Could you please put all these changes in the description for context?
